ledger/src/slot_stats.rs

```diff
@@ -115,20 +120,29 @@ impl SlotsStats {
                 slot_stats.last_index = meta.last_index.unwrap_or_default();
                 if !slot_stats.flags.contains(SlotFlags::FULL) {
                     slot_stats.flags |= SlotFlags::FULL;
-                    slot_full_reporting_info =
-                        Some((slot_stats.num_repaired, slot_stats.num_recovered));
+                    slot_full_reporting_info = Some((
+                        slot_stats.num_repaired,
+                        slot_stats.num_recovered,
+                        slot_stats.repaired_indexes.clone(),
+                    ));
                 }
             }
         }
         drop(stats);
-        if let Some((num_repaired, num_recovered)) = slot_full_reporting_info {
+        if let Some((num_repaired, num_recovered, repaired_indexes)) = slot_full_reporting_info {
             let slot_meta = slot_meta.unwrap();
             let total_time_ms =
                 solana_sdk::timing::timestamp().saturating_sub(slot_meta.first_shred_timestamp);
             let last_index = slot_meta
                 .last_index
                 .and_then(|ix| i64::try_from(ix).ok())
                 .unwrap_or(-1);
+            info!(
+                "Repair blockstore insertion metrics slot: {},
+                repaired_indexes: {:?}
+                ",
+                slot, repaired_indexes,
+            );
```

Comment:
In the wild:

```
[2022-06-17T04:07:30.145871000Z INFO  solana_ledger::slot_stats] Repair blockstore insertion metrics slot: 6,
                    repaired_indexes: {53, 63, 5, 54, 21, 4, 32, 14, 20, 41, 60, 16, 13, 19, 10, 2, 56, 7, 57, 30, 36, 59, 18, 47, 58, 62, 15, 9, 6, 24, 38, 55, 29, 52, 42, 40, 61, 50, 65, 48, 46, 25, 17, 34, 45, 12, 1, 22, 43, 64, 49, 23, 0, 11, 8, 27, 33, 51, 35, 3, 37, 31, 44, 28, 39, 26}
```

Reply:
@behzadnouri do you think it would be useful to log these FEC counts as well? https://github.com/solana-labs/solana/blob/9bc83075ebce893ff631bfc40901a8096e27c1ac/ledger/src/slot_stats.rs#L32. Seems like it would be helpful only if we knew how big the FEC sets were, can we assume 32:32 in most cases?

Reply:
yeah, it is always 32:32 except for the last batch which is `64 - num-data-shreds` coding shreds.